osu.Game/Graphics/UserInterface/FPSCounterTooltip.cs

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Linq;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Cursor;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Platform;
+using osu.Game.Graphics.Containers;
+using osuTK;
+
+namespace osu.Game.Graphics.UserInterface
+{
+    public class FPSCounterTooltip : CompositeDrawable, ITooltip
+    {
+        private OsuTextFlowContainer textFlow = null!;
+
+        [Resolved]
+        private GameHost gameHost { get; set; } = null!;
+
+        [BackgroundDependencyLoader]
+        private void load(OsuColour colours)
+        {
+            AutoSizeAxes = Axes.Both;
+
+            CornerRadius = 15;
+            Masking = true;
+
+            InternalChildren = new Drawable[]
+            {
+                new Box
+                {
+                    Colour = colours.Gray1,
+                    Alpha = 1,
+                    RelativeSizeAxes = Axes.Both,
+                },
+                new OsuTextFlowContainer(cp =>
+                {
+                    cp.Font = OsuFont.Default.With(weight: FontWeight.SemiBold);
+                    cp.Spacing = new Vector2(-1);
```

Comment:
Names read better without negative kerning (not sure if intended or not)
```suggestion
```

| before | after |
|--------|-------|
| <img width="273" alt="CleanShot 2022-07-21 at 02 54 10@2x" src="https://user-images.githubusercontent.com/22781491/180101975-35ccefe8-73cf-4c14-bee0-51b7893b5666.png"> | <img width="273" alt="CleanShot 2022-07-21 at 02 54 10@2x" src="https://user-images.githubusercontent.com/22781491/180101988-f6a90a1a-f17b-4fd8-9341-3fb9b2fb54c5.png"> |

Reply:
I previously had them set to fixed width where the negative spacing worked well. Not necessary anymore.